gradle-plugin/build.gradle

```diff
@@ -6,16 +6,13 @@ plugins {
 }
 apply plugin: 'kotlin'
 
-project.group = 'io.arrow-kt'
-project.version = '0.0.1'
 def pluginId = "arrow.meta.plugin.gradle"
 
 dependencies {
   implementation gradleApi()
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   implementation "org.jetbrains.kotlin:kotlin-gradle-plugin-api:$kotlin_version"
   implementation "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-  compileOnly project(":compiler-plugin") { "embeddable" }
```

Comment:
Why did you delete this? 😄

Reply:
Gradle plugin does not need compiler-plugin embedded - it's downloaded from repo